x/leverage/keeper/interest.go

```diff
@@ -73,7 +73,13 @@ func (k Keeper) AccrueAllInterest(ctx sdk.Context) error {
 
 	// calculate time elapsed since last interest accrual (measured in years for APR math)
 	if currentTime < prevInterestTime {
-		return types.ErrNegativeTimeElapsed.Wrap(fmt.Sprintf("current: %d, prev: %d", currentTime, prevInterestTime))
+		k.Logger(ctx).With("AccrueAllInterest will wait for block time > prevInterestTime").Error(
+			types.ErrNegativeTimeElapsed.Error(),
+			"current", currentTime,
+			"prev", prevInterestTime,
+		)
+
+		return nil
```

Comment:
Log is not enough. I think we should either return some message or keep the error.

Reply:
What do you think about [these options](https://github.com/umee-network/umee/issues/543#issuecomment-1152706616)?

Reply:
OK